srcpkgs/containers.storage/template

```diff
@@ -17,5 +18,6 @@ do_build() {
 
 do_install() {
 	vman "$_manpage"
+	vsed -e 's/^#mount_program/mount_program/' -i storage.conf
```

Comment:
```suggestion
	vsed -e 's/^#mount_program = "/mount_program = "/' -i storage.conf
```

I would either anchor this a bit more to avoid unintentionally uncommenting a descriptive line like

> #mount_program can be set to [...]

should it somebody add that upstream. Alternatively, carry an explicit patch.